android/app/src/main/java/app/covidshield/MainApplication.java

```diff
@@ -65,6 +67,8 @@ public ReactNativeHost getReactNativeHost() {
     @Override
     public void onCreate() {
         instance = this;
+        Log.d("locale", Locale.getDefault().getLanguage());
```

Comment:
The locale will need to come from the user selected and saved language, not the system locale.

Reply:
Calling that out of scope for this PR.

Sticking just to the XML generating scripts for this one.

Mentioned in the Trello card:
> Out of scope
Updating the app to set the default language in Kotlin when the user changes the device language within the app

Reply:
For another PR we should be able to use `Locale.setDefault(new Locale("fr_CA"));`

Reply:
@timarney @jeberhardt Here is how we can use the generated files https://github.com/cds-snc/covid-alert-app/commit/1585adb3fa667f5016e04e62fbf685f9eeb910d8

@timarney I did not add the commit in this PR as it uses code that is not yet been properly tested yet and is also a WIP

Reply:
I think we could merge this as is and do a follow up PR for the other commit.

Reply:
@timarney I do agree with you! The follow up PR should be based on the branch we are currently working on with @jeberhardt (https://github.com/cds-snc/covid-alert-app/tree/feature/android-exposure-notification-port) because it is where we will need to plug the language stuff.

Before reviewing the PR, would you mind squashing the commits? I feel like it is almost all about the same `translations-android.js` script